### DISCUSSION TITLE
How can I disable installation of `@prisma/client` and `prisma` when generating JavaScript client?

### DISCUSSION THREAD

**Discussion by alissa-tung** [prisma/prisma#25663] [Q&A] [answered]
### Question

How can I disable installation of `@prisma/client` and `prisma` when generating JavaScript client? Since I do not need them

**Comment by ludralph** [comment]
Hi @alissa-tung 

Do you mean you want to uninstall `@prisma/client` and `prisma` from your `package.json` file? If yes, if you're using npm, you can simply run the command below to remove both dependencies
```
npm uninstall @prisma/client
```
and 
```
```
npm uninstall prisma
```

**Reply by alissa-tung** [reply]
thank you. Yes, currently my Makefile contains `prisma generate && pnpm rm '@prisma/client' prisma && pnpm dedupe && pnpm i`, but I hope there is an option for not install them from the beginning

**Reply by ludralph** [reply, marked as answer]
Hi @alissa-tung 

We currently don't provide a built-in flag to prevent the installation of `@prisma/client` and `prisma` entirely from the start. You may however open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you think this could be a useful feature to add and document your usecase.

**Reply by alissa-tung** [reply]
Thank you
